app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileEditFragmentTest.kt

```diff
@@ -158,6 +166,36 @@ class ProfileEditFragmentTest {
     }
   }
 
+  @Test
+  fun testProfileEdit_startWithUserProfile_clickProfileDeletionButton_deleteSnackbarIsVisible() {
+    launchFragmentTestActivity(internalProfileId = 1).use {
+      onView(withId(R.id.profile_delete_button)).perform(click())
+      testCoroutineDispatchers.runCurrent()
+      onView(withText(R.string.profile_edit_delete_dialog_positive))
+        .inRoot(isDialog())
+        .perform(click())
+      testCoroutineDispatchers.runCurrent()
+      onView(allOf(withText(R.string.profile_edit_delete_success)))
+        .check(matches(isDisplayed()))
+    }
```

Comment:
You could try using the visibility matcher instead of the isDisplayedMatecher()

```suggestion
onView(withText(R.string.profile_edit_delete_success))
        .check(matches(ViewMatchers.withEffectiveVisibility(ViewMatchers.Visibility.VISIBLE)))
```

Reply:
Okay done.